src/stores/blueprints-store.ts

```diff
@@ -24,8 +27,11 @@ const useStore = create<StoreState>((set, get) => ({
       }),
     });
     const result = await response.json();
-    const blueprints = result.blueprints;
-    set({ blueprints });
+    set({ 
+      blueprints: result.blueprints,
+      paging: result.paging,
+    });
```

Comment:
last line I mentioned, this is sending tag counts to tag store so that it can filter the tags whenever we get a new query back